File: mandelbrot_py/mandelbrot_py/count2color.py
def count_to_color(count: int, max_count: int) -> tuple[int, int, int]:
    if count == max_count:
        return (0, 0, 0)  # 黒
    else:
        return (
            0,
            int((count / max_count) * 255),
            int((count / max_count) * 255)
        )

File: mandelbrot_rs/src/count2color.rs
pub fn count_to_color(cnt: usize, max_count: usize) -> [u8; 3] {
    if cnt == max_count {
        [0, 0, 0] // 黒
    } else {
        [
            0,
            ((cnt as f32 / max_count as f32) * 255_f32) as u8,
            ((cnt as f32 / max_count as f32) * 255_f32) as u8,
        ]
    }
}
